### DISCUSSION TITLE
Regarding the issue that colors are not displayed after importing Carla

### DISCUSSION THREAD

**Discussion by yubinling** [carla-simulator/carla#7310] [Q&A]
Hello, I want to ask a question, I used Roadrunner to draw a road, and pasted the png file QR code I imported myself on the road, as shown in Figure 1, as a custom flag, but after I imported Carla, the QR code could not be displayed, how to deal with this kind of problem
![Uploading 1260969260f672783162bc2dac75f90.jpg…]()

**Comment by RodrigoVillarias** [comment]
Hi! could you upload the image again? it seems that it didn't upload and the link is broken
